src/test/java/org/jenkinsci/test/acceptance/junit/WithSystemPropertiesTest.java

```diff
@@ -0,0 +1,36 @@
+package org.jenkinsci.test.acceptance.junit;
+
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.arrayContainingInAnyOrder;
+import static org.hamcrest.Matchers.emptyArray;
+import static org.junit.Assert.assertNull;
+
+public class WithSystemPropertiesTest {
+
+    @Test
+    public void combine_WithPlugins_annotations() throws Exception {
+        assertThat(WithSystemProperties.RuleImpl.getProperties(FakeTestClass.class.getAnnotation(WithSystemProperties.class))
```

Comment:
this smells to me.  these are not properties they are property string arguments (ie they have -D).